packages/wrangler/src/__tests__/cloudchamber/apply.test.ts

```diff
@@ -135,9 +135,6 @@ describe("cloudchamber apply", () => {
 			│   [containers.constraints]
 			│   tier = 2
 			│
-			├ Do you want to apply these changes?
```

Comment:
it seems that all these tests test the non-interactive mode. Is there a reason why interactivity isn't tested for this cmd?